contributing.md

```diff
@@ -0,0 +1,67 @@
+Contributing
+============
+
+## Issue Tracking
+
+New feature requests, changes, enhancements, non-methodology features, and bug reports can be filed as new issues in the
+[GitHub.com issue tracker](https://github.com/entralliance/entr_runtime/issues) at any time. Please be sure to fully describe the
+issue.
+
+<!---For other issues, please email the ENTR distribution list at `xx`.--->
+
+## Repository
+
+The entr_runtime repository is hosted on GitHub, and located here: https://github.com/entralliance/entr_runtime
```

Comment:
we may want to have some clarifying guidance on whether to open an issue in the runtime or warehouse repo - it also could be worth considering having a project-level repo again that's just for project mgmt, issue triage, etc.

Reply:
i think this is an excellent point. do we have a good sense of the system components now? perhaps some sort of diagram that we can include?